src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -9009,7 +9012,8 @@ static bool isCandidateUnique(ref MethodSymbol? method, MethodSymbol candidate)
             ImmutableArray<RefKind> parameterRefKinds,
             ImmutableArray<DeclarationScope> parameterScopes,
             ImmutableArray<TypeWithAnnotations> parameterTypes,
-            ImmutableArray<ConstantValue?> parameterDefaultValues)
+            ImmutableArray<ConstantValue?> parameterDefaultValues,
+            ImmutableArray<SourceComplexParameterSymbolBase> parameterSymbolsForAttributes)
```

Comment:
>parameterSymbolsForAttributes [](http://example.com/codeflow?start=61&length=29)

Could we pass method symbol instead of the last 7 parameters and have this function extract necessary information directly from it?